packages/api/src/utils/db-client.js

```diff
@@ -83,13 +83,15 @@ export class DBClient {
     magic_link_id,
     github_id,
     did,
-    keys:auth_key_user_id_fkey(user_id,id,name,secret)
+    keys:auth_key_user_id_fkey(user_id,id,name,secret),
```

Comment:
some kind of editor or linter support for this would be nice. No idea if that exists, you you need an eagle eye to catch this